### ISSUE TITLE
Update press sheet

### ISSUE THREAD

**Issue by LauraUSDS** [usds/website#516]
Is currently the 2019 congressional one-sheet. Make a slightly-tweaked version of the recruiting one-sheet (see details on https://github.com/usds/usds/issues/4956). 

Will need to update wording from "press kit" to "press sheet." This applies to the 'news&blog' page, the footer...where else?

Timing: as able.

**Comment by LauraUSDS** [comment]
@drew-usds When you have a moment (~10 min), I'd love your help with this! 

I've got the new press sheet ready to go; GH won't let me upload files here or to the website repo, but I did mange to [get it into the branding repo](https://github.com/usds/branding/blob/master/One-Sheets/1_USDS/2020-03-Press-1-Sheet/USDS-Press-Sheet.pdf) by sheer brute force.

It has the same name as the existing press sheet, so we just need to overwrite the file in [website > assets > files](https://github.com/usds/website/tree/master/assets/files).

I'd also like to change the display text from 'press kit' to 'press sheet.' Does that phrase appear anywhere else besides the footer and in the 'news&blog' section? I'm guessing it's easy to make this change...I just don't know how and would like your help. 

Many thanks!
